src/otx/algorithms/anomaly/tasks/inference.py

```diff
@@ -302,6 +303,9 @@ def export(
             subprocess.run(optimize_command, check=True)
             bin_file = glob(os.path.join(self.config.project.path, "*.bin"))[0]
             xml_file = glob(os.path.join(self.config.project.path, "*.xml"))[0]
+
+            self._convert_to_new_format(xml_file)
```

Comment:
Even though this is a temporary solution, I'm not sure about the naming as `new` could be subjective and become old quickly. 

We could perhaps have the following alternative namings:

1. `self.convert_to_v*_format`
2. `self.add_metadata_to_model_ir`
3. `self.add_metadata_to_ir`
4. `self.append_metadata_to_model_ir`
5. `self.append_metadata_to_ir`